frontend/src/App.tsx

```diff
@@ -124,6 +125,11 @@ const App = (): React.ReactElement => {
                   path={Routes.ADMIN_FOOD_RESCUE_EDIT_DESCRIPTION_PAGE}
                   component={EditFoodRescueDescriptionPage}
                 />
+                <PrivateRoute
+                  exact
```

Comment:
can we add `adminOnly` prop here so that donors and volunteers cant access this route